core/src/main/scala/filodb.core/memstore/OnDemandPagingShard.scala

```diff
@@ -223,11 +223,10 @@ TimeSeriesShard(ref, schemas, storeConfig, shardNum, bufferMemoryManager, rawSto
         case req: TimeRangeChunkScan      =>  if (partition.numChunks > 0) {
                                                 val memStartTime = partition.earliestTime
                                                 if (req.startTime < memStartTime && partStartTime < memStartTime) {
-                                                  val toODP = TimeRangeChunkScan(req.startTime, memStartTime - 1)
+                                                  val toODP = TimeRangeChunkScan(req.startTime, memStartTime)
```

Comment:
Maybe add a comment about the boundary condition and explain what bug could result if we used -1?

Reply:
The CQL to fetch chunks uses `<` operator, and not `<=` operator on the endTime. Hence `-1` is not needed.